src/reducers/rootReducer.js

```diff
@@ -1,5 +1,9 @@
 import { combineReducers } from 'redux';
+import modal from './modal';
 
-const rootReducer = combineReducers({});
+const appReducer = combineReducers({ modal });
 
+// const initialState = appReducer({}, {});
+
+export const rootReducer = (state, action) => appReducer(state, action);
```

Comment:
Since you are already exporting rootReducer as a default export, perharps you don't need to export it as a named export on this line. Check to confirm that I'm in line with your thought process.

Reply:
Thanks @henperi, this was erroneously left out.

Reply:
Resolved.